client/asset/eth/nodeclient_harness_test.go

```diff
@@ -135,6 +135,8 @@ var (
 	// funds will be printed.
 	testnetWalletSeed            string
 	testnetParticipantWalletSeed string
+	usdcID, _                    = dex.BipSymbolID("usdc.eth")
```

Comment:
Okay for now. I'd like to see a little cmd to load the wallet (can use the same key file) and run `GetGasEstimates`. My reasoning is this. We designed the gas system so that an operator can adjust the gases up to a factor of 2, in case some unforeseen subtlety of an ERC20 contract causes gas to exceed our initial expectations. But in the case that an operator recognizes a problem, it would be prudent to run some tests to see what values are popping out, and to see if other estimates need adjusting as well.

Reply:
A cli tool to run `GetGasEstimates` sounds smart to me.  I'm also not fond of functions like that in the product code that are not intended to be used in the app.

However, we no longer do operator-set gasses (or transaction sizes for utxo assets).  Clients only use their own baked-in trusted values.  So if we want it back so that an operator can "adjust the gases up to a factor of 2, in case some unforeseen subtlety of an ERC20 contract", we'll have to modify the `asset.Wallet` interface again.
But I think we accepted, that a server changing a client's understanding of transaction weight (not just fee rate) is not a desirable approach, especially in light of mesh.

IMO we should pad the compiled-in gasses more generously.  Consider: If you use metamask to interact with these contracts manually, it sets a surprisingly high gas limit for the transaction.  See this comment regarding the `approve` method:  https://github.com/decred/dcrdex/pull/1733#discussion_r1035492946  I don't know how it gets the gas limit number (some evm simulation?), but I think it's an indication that we should pad these numbers more generously.  In that limited test, it sets the gas limit to 150% of what the executed transaction ends up actually consuming.